src/state_types/models/context.rs

```diff
@@ -28,12 +29,28 @@ pub struct Auth {
 pub struct CtxContent {
     pub auth: Option<Auth>,
     pub addons: Vec<Descriptor>,
+    pub settings: HashMap<String, String>,
 }
 impl Default for CtxContent {
     fn default() -> Self {
+            let mut settings = HashMap::new();
```

Comment:
you should use `maplit` or `vec![("language", "eng"), ...].iter().cloned().collect()`